code/modules/stock_market/articles.dm

```diff
@@ -0,0 +1,125 @@
+		"expand_influence" = list("expands their influence over", "continues to dominate", "looks to gain shares in", "rolls their new product line out in"), \
+		"failure" = list("failure", "meltdown", "breakdown", "crash", "defeat", "trainwreck", "wreck"), \
+		"famous" = list("famous", "prominent", "leading", "renowned"), \
+		"hit_shelves" = list("hit the shelves", "appeared on the market", "came out", "was released"), \
+		"industry" = list("industry"), \
+		"industrial" = list("industrial"), \
+		"jobs" = list("workers"), \
+		"negative_outcome" = list("it's not leaving the shelves", "nobody seems to have taken note", "no notable profits have been reported", "it's beginning to look like a huge failure"), \
+		"neutral_outcome" = list("it's not lifting off as expected", "it's not selling according to the expectations", "it's only generating enough profit to cover the marketing and manufacturing costs", "it does not look like it will become a massive success"), \
+		"positive_outcome" = list("it's already sold out", "it has already sold over one billion units", "suppliers cannot keep up with the wild demand", "several companies using this new technology are already reporting a projected increase in profits"), \
+		"resounding" = list("resounding", "tremendous", "total", "massive", "terrific", "colossal"), \
+		"rise" = list("rise", "increase", "fly off the positive side of the charts", "skyrocket", "lift off"), \
+		"sell" = list("sell!", "sell, sell, sell!", "bail!", "abandon ship!", "get out before it's too late!", "evacuate!", "withdraw!"), \
+		"signifying" = list("signifying", "indicating", "displaying the sign of", "displaying"), \
+		"sneak_peek" = list("review", "sneak peek", "preview"), \
+		"stock_market" = list("stock market", "stock exchange"), \
+		"stockholder" = list("stockholder", "shareholder"), \
+		"success" = list("success", "triumph", "victory"), \
+		"this_time" = list("this week", "last week", "this month", "yesterday", "today", "a few days ago") \
+	)
+
+/datum/article/New()
+	..()
+	if ((outlets.len && !prob(100 / (outlets.len + 1))) || !outlets.len)
+		var/ON = generateOutletName()
+		if (!(ON in outlets))
+			outlets[ON] = list()
+		outlet = ON
+	else
+		outlet = pick(outlets)
+
+	var/list/authors = outlets[outlet]
+	if ((authors.len && !prob(100 / (authors.len + 1))) || !authors.len)
+		var/AN = generateAuthorName()
+		outlets[outlet] += AN
+		author = AN
+	else
+		author = pick(authors)
+
+	ticks = world.time
+
+/datum/article/proc/generateOutletName()
+	var/list/locations = list("Earth", "Luna", "Mars", "Saturn", "Jupiter", "Uranus", "Pluto", "Europa", "Io", "Phobos", "Deimos", "Space", "Venus", "Neptune", "Mercury", "Kalliope", "Ganymede", "Callisto", "Amalthea", "Himalia")
+	var/list/nouns = list("Post", "Herald", "Sun", "Tribune", "Mail", "Times", "Journal", "Report")
+	var/list/timely = list("Daily", "Hourly", "Weekly", "Biweekly", "Monthly", "Yearly")
```

Comment:
Add Parsecly. As a reference to Starwars. (Yes, I know a parsec a distance not a unit of time).